### ISSUE TITLE
Fails to build with --without-dnet

### ISSUE THREAD

**Issue by acepace** [vmware/open-vm-tools#178]
Clean install with --without-x and --without-dnet (which doesn't work) leads to 
```

nicInfoPosix.c:986:1: error: ‘ValidateConvertAddress’ defined but not used [-Werror=unused-function]
 ValidateConvertAddress(const struct sockaddr *addr)  // IN:
 ^~~~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
```

Which obviously leads to the build failing.

Warning should either be fixed or Werror removed from this file.

**Comment by acepace** [comment]
This is fixed by removing --without-dnet, which considering [issue#61](https://github.com/vmware/open-vm-tools/issues/61), leads me to suggest removing this config flag.

**Comment by oliverkurth** [comment]
Thank you for this feedback!

**Comment by oliverkurth** [comment]
Issue is resolved with commit 009def580b00040fa6e6c07cb5743e60973973f5
